pkg/action/cl001/ac000/plan.go

```diff
@@ -36,17 +36,17 @@ var Plan = []plan.Step{
 	},
 	{
 		Action:  "ac004",
-		Backoff: plan.NewBackoff(24*time.Minute, 3*time.Minute),
+		Backoff: plan.NewBackoff(30*time.Minute, 3*time.Minute),
 		Comment: "check worker count",
 	},
 	{
 		Action:  "ac006",
-		Backoff: plan.NewBackoff(10*time.Second, 2*time.Second),
+		Backoff: plan.NewBackoff(15*time.Minute, 1*time.Minute),
 		Comment: "check master host network",
 	},
 	{
 		Action:  "ac007",
-		Backoff: plan.NewBackoff(10*time.Second, 2*time.Second),
+		Backoff: plan.NewBackoff(15*time.Minute, 1*time.Minute),
```

Comment:
Here waiting 15 minutes again does not make sense to me. We waited already above.

Reply:
Same reason as for the master host network check. I noticed that kiam is coming up really late on the node, so the reason is basically I wanted to wait until all pods are running.

Reply:
The thing is that this is compounded here already. I get the first 15 minutes. I do not get the second 15 minutes. Why does it take half an hour for all pods to be scheduled?

Reply:
Reverted 👍